### PR TITLE
ability of posting comments added

### PR DESCRIPTION
**Pull request by safchik**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -40,10 +40,20 @@ export const fetchComments = (article_id) => {
         });
 }
 
+export const postComment = (article_id, comment) => {
+    console.log(comment)
```
Comment (popatre) [inline src/api.js]: console log - be gone 🚮

File: src/components/Comments.jsx [hunk 2]
```diff
@@ -14,15 +22,50 @@ function Comments({ article_id }) {
         });
     }, [article_id]);
 
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        console.log(newComment, user)
```
Comment (popatre) [inline src/components/Comments.jsx]: console log - be gone 🚮

File: src/components/Comments.jsx [hunk 3]
```diff
@@ -14,15 +22,50 @@ function Comments({ article_id }) {
...
+        postComment(article_id, { body: newComment, username: user.username })
+            .then((response) => {
+                setComments([...comments, response]);
```
Comment (popatre) [inline src/components/Comments.jsx:30]: Would be better for the new comment to appear at the top

File: src/components/Comments.jsx [hunk 4]
```diff
@@ -14,15 +22,50 @@ function Comments({ article_id }) {
...
         setExpanded(!isExpanded);
     };
 
     return (
         <section id="comments">
+            <div>
+                <form onSubmit={handleSubmit}>
+                    <label>
+                        Your comment:
+                        <textarea value={newComment} onChange={(event) => setNewComment(event.target.value)} />
```
Comment (popatre) [inline src/components/Comments.jsx]: Nice controlled component

**Review by popatre — CHANGES_REQUESTED** [review]
Just a few things to add:

- [x] I crash the whole app if i try and post a comment when not signed in
- [x] Im also not sure how to sign in - nothing seems to work
- [x] Would be better for the comment ti appear at the top

Could the fix the login issue and re-submit? I couldnt properly test it as i couldnt actually post anything

File: src/components/Comments.jsx (hunk 3 above)
Comment (safchik) [reply src/components/Comments.jsx:30]: solved

**Review by popatre — APPROVED** [review]
Looks all good, well done 👍

**Pull request merged**